% Template for PLoS
% Version 1.0 January 2009
%
% To compile to pdf, run:
% latex plos.template
% bibtex plos.template
% latex plos.template
% latex plos.template
% dvipdf plos.template

\documentclass[10pt]{article}

% amsmath package, useful for mathematical formulas
\usepackage{amsmath}
% amssymb package, useful for mathematical symbols
\usepackage{amssymb}

% graphicx package, useful for including eps and pdf graphics
% include graphics with the command \includegraphics
\usepackage{graphicx}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

\usepackage{color} 

% Use doublespacing - comment out for single spacing
%\usepackage{setspace} 
%\doublespacing


% Text layout
\topmargin 0.0cm
\oddsidemargin 0.5cm
\evensidemargin 0.5cm
\textwidth 16cm 
\textheight 21cm

% Bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,justification=raggedright]{caption}

% Use the PLoS provided bibtex style
\bibliographystyle{plos2009}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother


% Leave date blank
\date{}

\pagestyle{myheadings}
%% ** EDIT HERE **


%% ** EDIT HERE **
%% PLEASE INCLUDE ALL MACROS BELOW

%% END MACROS SECTION

\begin{document}

% Title must be 150 characters or less
\begin{flushleft}
{\Large
\textbf{Selection or Drift on Repetitive Elements Causes Large Genome Size Variation in \emph{Zea} and \emph{Tripsacum}}
}
% Insert Author names, affiliations and corresponding author email.
\\
Paul Bilinski$^{1}$, 
Anne Lorant$^{1}$
Author$^{1}$, 
Jeffrey Ross-Ibarra$^{1,2,\ast}$
\\
\bf{1} Dept. of Plant Sciences, University of California, Davis, CA, USA
\\
\bf{2} Author2 Dept/Program/Center, Institution Name, City, State, Country
\\
\bf{3} Author3 Dept/Program/Center, Institution Name, City, State, Country
\\
$\ast$ E-mail: rossibarra@ucdavis.edu
\end{flushleft}

% Please keep the abstract between 250 and 300 words
\section*{Abstract}

Structural variation is rampant in the maize genome and is known to contribute greatly to large fluctuations in genome size, though little is known as to how selection acts on broad structural variation.
Using next-gen sequence data, we investigate genome-wide changes in repetitive content in maize landraces, teosinte, and \emph{Zea}'s sister genus \emph{Tripsacum} across an altitudinal gradient in Mexico and South America to identify the repetitive content responsible for shifting genome sizes.
We also use models akin to $F_{st}$ - $Q_{st}$  analysis to search for evidence of selection across populations, treating repetitive content as our phenotype.
%We focus on transposable elements (TEs), heterochromatic knobs, and tandem centromeric repeats because together they can account for a vast majority of the genome.  
We corrected for repetitive content abundance using genome size measures, and found incredible variation in individual repetitive tags across our altitudinal gradient.
%We find a number of environmental trends, the strongest being a correlation between altitude and genome size, as has been previously documented.  
Surprisingly, different classes of repeats showed opposing clinal patterns after correcting for genetic relatedness.
%Is it selection or no
%do we see differences in maize trip, is it parallel
%big take away

% Please keep the Author Summary between 150 and 200 words
% Use first person. PLoS ONE authors please skip this step. 
% Author Summary not valid for PLoS ONE submissions.   
\section*{Author Summary}

\section*{Introduction}

Last: table of taxa we are using


% Results and Discussion can be combined.
\section*{Results}

\subsection*{Genome Size Variation and Repeats}

Plot of genome size var in all taxa

\subsection*{Subsection 2}

\section*{Discussion}

% You may title this section "Methods" or "Models". 
% "Models" is not a valid title for PLoS ONE authors. However, PLoS ONE
% authors may use "Analysis" 
\section*{Materials and Methods}

\subsection*{Plant Material}

\subsection*{Sequencing}
snp chip data
Skim Sequencing
GBS of trip

\subsection*{Genome Size Estimates}
Absolute genome size measures were performed by Plant Cytometry Services (JG Schijndel, NL).
All maize and teosinte samples were grown in greenhouse conditions, and leaf samples were collected and sent for measure at the same time.
For populations of teosinte, two individuals per population were measured as previous studies have shown the greatest genome size variation across populations \cite{diez2013genome}, while genome size measures were performed individually on each maize landrace accession.
\emph{T. dactyloides} experienced low germination rates, and therefore we were only able to perform genome size measure on XX individuals.
These individuals spanned the altitudinal range of our \emph{T. dactyloides} samples.
We sequenced to low coverage our XX germinated \emph{T. dactyloides} individuals and observed a strong correlation between genome size measures and relative genomic content mapping to maize cDNA (ref{}).
To expand the number of \emph{T. dactyloides} in our study, we extracted DNA from single embryos and performed low coverage sequencing and genotyping by sequencing (GBS).
We used total genomic content mapping to maize cDNA to estimate relative genome size of our 96 \emph{T. dactyloides} embryos.

\subsection*{Repetitive Content Mapping}

Mapping references and sequence ID's were constructed separately each repeat and are available on our github repository (\emph{https://github.com/paulbilinski/}). %\url{Github_GS_analyses}
Reference sequences for canonical knobs, TR-1 knobs, and rDNA repeats were gathered from NCBI, and sequence identifiers are available in the reference files on github.
We used the library of CentC repeats assembled in cite{}.
Chloroplast DNA and cDNA were taken from the maize reference genome (5b, www.maizesequence.org).
For the transposable element database, we began with the TE database consensus sequences from \cite{baucom2009exceptional}.
We BLASTed sequences against themselves and masked shared regions, only retaining unique tags that were at least 70bp in length in our mapping reference.

Read mapping was performed using the Burrows-Wheeler Aligner \cite{li2009fast}.
For repetitive sequences, we used parameters -B 2 -k 11 -a to store all hit locations with an identity threshold of approximately 80\%.
We decided upon a minimum seed length of 11 as it produced the most reads mapping against the full transposable element database.
To test whether alignment algorithm played a large role in our repetitive content measures, we compared measures from BWA to measures from Mosaik, used for previously repetitive content mapping in cite{}.
We observed nearly perfect correlations between total 180bp knob content across the panel with the two aligners ref{}.
%We also tested transposable element abundance in a single, random individual between the two aligner, and observed a strong correlation.
All data for this study was generated with BWA due to its speed of alignment and broader use in the community.

\subsection*{Genotype-Phenotype Evolution}
Black box of ovaskienen.

% Do NOT remove this, even if you are not including acknowledgments
\section*{Acknowledgments}
shout out to the homeys.

%\section*{References}
% The bibtex filename
\bibliography{refgs.bib}

\section*{Figure Legends}
%\begin{figure}[!ht]
%\begin{center}
%%\includegraphics[width=4in]{figure_name.2.eps}
%\end{center}
%\caption{
%{\bf Bold the first sentence.}  Rest of figure 2  caption.  Caption 
%should be left justified, as specified by the options to the caption 
%package.
%}
%\label{Figure_label}
%\end{figure}


\section*{Tables}
%\begin{table}[!ht]
%\caption{
%\bf{Table title}}
%\begin{tabular}{|c|c|c|}
%table information
%\end{tabular}
%\begin{flushleft}Table caption
%\end{flushleft}
%\label{tab:label}
% \end{table}

\end{document}

